nixos/modules/services/x11/desktop-managers/gnome.nix

```diff
@@ -357,6 +357,8 @@ in
         isSystem = true;
       };
 
+      services.orca.enable = mkDefault true;
```

Comment:
You can use `notExcluded pkgs.orca` to preserve the behaviour.

Reply:
Thanks, done.